src/main/kotlin/no/nav/familie/oppdrag/konsistensavstemming/KonsistensavstemmingMapper.kt

```diff
@@ -91,12 +91,21 @@ class KonsistensavstemmingMapper(private val fagsystem: String,
 
     private fun akkumulerTotalbeløp(utbetalingsperiode: Utbetalingsperiode) {
         // utlede om utbetalingsperioden er aktuell for avstemmingsdato
-        if (utbetalingsperiode.vedtakdatoFom.isBefore(avstemmingsDato.toLocalDate()) && utbetalingsperiode.vedtakdatoTom.isAfter(avstemmingsDato.toLocalDate())) {
+        if (erPeriodenAktiv(utbetalingsperiode)) {
             totalBeløp+=utbetalingsperiode.sats.toLong()
             totalantall++
         }
     }
 
+    private fun erPeriodenAktiv(utbetalingsperiode: Utbetalingsperiode): Boolean {
+        if (utbetalingsperiode.vedtakdatoFom.isBefore(avstemmingsDato.toLocalDate()) && utbetalingsperiode.vedtakdatoTom.isAfter(avstemmingsDato.toLocalDate())) {
+            return true
+        } else if (utbetalingsperiode.vedtakdatoFom.isAfter(avstemmingsDato.toLocalDate())) {
+            return true
+        }
+        return false
+    }
+
```

Comment:
KUNNE kanskje skrives som:

```
    val vedtakdatoFom = utbetalingsperiode.vedtakdatoFom;
    val vedtakdatoTom = utbetalingsperiode.vedtakdatoTom;
    val lokalAvstemmingsdato = avstemmingsDato.toLocalDate()

    return (vedtakdatoFom < lokalAvstemmingsdato && vedtakdatoTom > lokalAvstemmingsdato)
           || vedtakdatoFom > lokalAvstemmingsdato
```
eller no' sånt

Reply:
Hehe ja, enig i at det var ryddigere! Takk for godt forslag :)